KeyDistribution_E91/Tasks.qs

```diff
@@ -0,0 +1,178 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+namespace Quantum.Kata.KeyDistributionE91 {
+
+    open Microsoft.Quantum.Arrays;
+    open Microsoft.Quantum.Measurement;
+    open Microsoft.Quantum.Canon;
+    open Microsoft.Quantum.Intrinsic;
+    open Microsoft.Quantum.Diagnostics;
+    open Microsoft.Quantum.Convert;
+    open Microsoft.Quantum.Math;
+    open Microsoft.Quantum.Random;
+
+
+
+    //////////////////////////////////////////////////////////////////
+    // Part I. Preparation
+    //////////////////////////////////////////////////////////////////
+	
+    // Task 1.1. Entangled Pairs
+    // Inputs:
+    //      1) "qsAlice": an array of N qubits in the |0⟩ states,
+    //	    2) "qsBob": an array of N qubits in the |0⟩ states,
+    // Goal: Create entangled pairs |Ψ⟩ = 1/√2 (|00⟩ + |11⟩) out of corresponding qubits of
```

Comment:
Which protocol description are you following? The one I find clearest is https://ux1.eiu.edu/~nilic/Nina's-article.pdf, and it suggests that the pair should be 1/√2 (|01⟩ - |10⟩), which then should yield anti-correlated results for matching bases choice, unlike in BB84 protocol.